docker-compose.windows.yml

```diff
@@ -1,16 +1,16 @@
-version: '2'
+version: '3'
 
 services:
+  puppet:
+    networks:
+      default:
+        aliases:
+         - puppet.${AZURE_DOMAIN}
+
   postgres:
+    image: stellirin/postgres-windows:9.6
+    environment:
+      - PGDATA=c:\data
     volumes:
-      - puppetdb-postgres-volume:/var/lib/postgresql/data/
-
-# An external volume is needed for postgres as when under Docker For Windows the files have a different
-# owner that the starting process.
-# ref: https://forums.docker.com/t/trying-to-get-postgres-to-work-on-persistent-windows-mount-two-issues/12456/4
-#
-# To create the volume use; docker volume create --name puppetdb-postgres-volume -d local
-# To delete the volume use; docker volume rm puppetdb-postgres-volume
-volumes:
-  puppetdb-postgres-volume:
-    external: true
+      - ./volumes/puppetdb-postgres/data:c:\data
+      - ./postgres-custom:c:\docker-entrypoint-initdb.d
```

Comment:
Should `postgres-custom` be under `./volumes/` ?

Reply:
Didn't you mention those container directories are empty by default? This is the "default" image.

Reply:
Yeah I'm just saying we can map in just that file and be we'd be safe if the default ever changes to include other files in that directory. 

Although that directory might be intended only for user extensions and will never include files by default.

Not a big deal, just mentioning it.

Reply:
IIRC there's a windows bug where you can only map directories...

Reply:
https://github.com/moby/moby/issues/30555

Reply:
> IIRC there's a windows bug where you can only map directories...

Very good to know if true....that could make things harder